webcomponents/bundle/src/main/bundle/calendar-wc.properties

```diff
@@ -1,3 +1,4 @@
 widget_title=Calendar
-pinned_sites_message=Showing all events for your pinned sites
+pinned_sites_message=Showing events from one year ago to a year from now, for your pinned sites
```

Comment:
```suggestion
pinned_sites_message=Showing events for the next year, for pinned sites only
```

Reply:
Are you not showing events for a 2 year window?

// Create a time range from one year ago to one year from now
        Instant oneYearAgo = now.minus(365, ChronoUnit.DAYS);
        Instant oneYearLater = now.plus(365, ChronoUnit.DAYS);

        return m_timeService.newTimeRange(oneYearAgo, oneYearLater);

Reply:
yes but most people just care about their calendar going forward not backward

Reply:
Fair enough.